.editorconfig

```diff
@@ -0,0 +1,13 @@
+﻿[*.cs]
+
+# CS1573: Parameter has no matching param tag in the XML comment (but other parameters do)
+dotnet_diagnostic.CS1573.severity = none
```

Comment:
Don't like this. If we're commenting public method we should always put all params in the comments

Reply:
removed

Reply:
Finally I suppressed those errors in-source.